app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from app.players import player
+from abc import ABC
+
+
+class Elf(player.Player, ABC):
+    def __init__(self, nickname: str,
+                 musical_instrument: str) -> None:
+        super().__init__(nickname)
+        self._musical_instrument = musical_instrument
+
+    def play_elf_song(self) -> None:
+        print(f"{self.nickname} is playing a song "
+              f"on the {self._musical_instrument}")
```

Comment:
The `Elf` class currently lacks the `get_rating` and `player_info` methods. For consistency with other player classes like `Dwarf`, consider implementing these methods to provide a rating and player information for elves as well.